src/App/Pages/Generator/GeneratorPageViewModel.cs

```diff
@@ -140,6 +158,12 @@ public bool IsUsername
             set => SetProperty(ref _isUsername, value);
         }
 
+        public bool IosExtension
+        {
+            get => _iosExtension;
+            set => SetProperty(ref _iosExtension, value);
+        }
```

Comment:
Do you need this to fire property changes? If not, remove the backing field as well